### DISCUSSION TITLE
Docx to Pdf Fails for JFIF Image Conversion

### DISCUSSION THREAD

**Discussion by mmurshed** [jgm/pandoc#9954] [Q&A]
The attached docx has a JFIF file but the conversion fails on my mac running Mac OS Sonoma 14.5 on M1 Macbook pro, pandoc 3.2.1. How can I convert docx containing JFIF? Docx file attached.

What's more confusing is that the docx was generated using pandoc from an html file. The html file contained and embedded JPG, which pandoc converted to JFIF in the docx. And now it can't open it back up.

**Command:** `pandoc --from docx --to pdf --pdf-engine=pdflatex -o 1.pdf 1.docx`
```
! LaTeX Error: Unknown graphics extension: .jfif.

l.91 ...ex2pdf.-d7b8394bfa43bf2f/media/rId25.jfif}
```

**Command:** `pandoc --from docx --to pdf --pdf-engine=xelatex -o 1.pdf 1.docx`

```
! LaTeX Error: Cannot determine size of graphic in /private/var/folders/4j/znth
913j3kd3f_dn4zg6bn_m0000gn/T/tex2pdf.-8d91215db68b9dd4/media/rId25.jfif (no Bou
ndingBox).
```

**Comment by jgm** [comment]
This error is coming from xelatex/pdflatex.  pdflatex can't handle jfif.  Maybe xelatex can, but it's having trouble determining the size. All of this is outside of pandoc's control.

More puzzling is why pandoc would have converted an embedded jpeg to jfif.  But to comment on that I'd need to see the html you converted from...

**Reply by mmurshed** [reply]
Thanks! When I ran pandoc to convert from html, the docx now has the jpg and converts correctly. So weird! It's working now end to end. I will let you know if I can repro. I removed the docx from the post for now.
